web/src/components/MediaControl/MediaControl.scss

```diff
@@ -18,12 +22,23 @@
 }
 
 .media-control {
-  font-size: 3.5rem;
   color: general.$controls-color;
   padding-left: 1.5rem;
   padding-right: 1.5rem;
 }
 
+@media screen and (max-width: 320px) {
+  .media-control {
+    font-size: var(--min-font-size);
+  }
+}
+
+@media screen and (min-width: 360px) {
```

Comment:
why are these two screen sizes different? perhaps I just don't understand CSS.

Reply:
That's because the space between is theoretically a space where it would transform percents at a time between the min and the max when I was trying to set the max to be the original 3.5 rem for a PC screen and 1 rem for the smallest of small, ideally keeping somewhere around the 2.5 rem that worked for most phones somewhere inbetween

That wasn't working so I made the max be 2.5 and so the scaling by percents here is kinda vestigial and unimportant, will remove just for the sake of clarity tomorrow